core/src/main/java/com/datastax/oss/driver/api/core/tracker/RequestTracker.java

```diff
@@ -52,20 +68,23 @@ default void onSuccess(
    * @param executionProfile the execution profile of this request.
    * @param node the node that returned the successful response.
    * @param requestLogPrefix the dedicated log prefix for this request
+   * @param executionInfo the execution info containing the results of this request
    */
   default void onSuccess(
       @NonNull Request request,
       long latencyNanos,
       @NonNull DriverExecutionProfile executionProfile,
       @NonNull Node node,
-      @NonNull String requestLogPrefix) {
+      @NonNull String requestLogPrefix,
+      @NonNull ExecutionInfo executionInfo) {
     // If client doesn't override onSuccess with requestLogPrefix delegate call to the old method
-    onSuccess(request, latencyNanos, executionProfile, node);
+    onSuccess(request, latencyNanos, executionProfile, node, requestLogPrefix);
```

Comment:
Note: even though onSuccess(request, latencyNanos, executionProfile, node, requestLogPrefix) is deprecated we still explicitly call it here (and take the hit of an extra method call that we don't really have to make) because some users may have overridden this method in their own RequestTracker impls and we want those impls to continue to work.